docs/source/index.rst

```diff
@@ -18,7 +18,7 @@ Synchronous client.
       from aleph_alpha_client import Client, CompletionRequest, Prompt
       import os
 
-      client = Client(token=os.getenv("AA_TOKEN"))
+      client = Client(token=os.getenv("AA_TOKEN"), host="https://inference-api.your-domain.com")
```

Comment:
Why not get the host from the .env, too?

Reply:
good catch. Probably the better approach, will update.

Reply:
done